engine/access/rpc/backend/backend_events.go

```diff
@@ -149,3 +168,29 @@ func verifyAndConvertToAccessEvents(execEvents []*execproto.GetEventsForBlockIDs
 
 	return results, nil
 }
+
+func (b *backendEvents) getEventsFromAnyExeNode(ctx context.Context, execNodes flow.IdentityList, req execproto.GetEventsForBlockIDsRequest) (*execproto.GetEventsForBlockIDsResponse, error) {
+	var errors *multierror.Error
+	// try to execute the script on one of the execution nodes
+	for _, execNode := range execNodes {
+		resp, err := b.tryGetEvents(ctx, execNode, req)
+		if err == nil {
+			return resp, nil
```

Comment:
do we have a logger on backendEvents, It'd be nice to print out which exe node we got this result from.

Reply:
don't have a logger in the backend - will add it. It needs to be passed in from the engine